Jenkinsfile

```diff
@@ -1,4 +1,8 @@
 buildPlugin(configurations: [
   [ platform: "linux", jdk: "8", jenkins: null ],
-  [ platform: "windows", jdk: "8", jenkins: null ]
-])
+  [ platform: "windows", jdk: "8", jenkins: null ],
+  [ platform: "linux", jdk: "8", jenkins: "2.150.1", javaLevel: "8" ],
+  [ platform: "windows", jdk: "8", jenkins: "2.150.1", javaLevel: "8" ],
+  [ platform: "linux", jdk: "11", jenkins: "2.156", javaLevel: "8" ],
```

Comment:
better to use 2.164 which will become lts soon

Reply:
sure